### ISSUE TITLE
Wildcard route is overwriting blueprint routes

### ISSUE THREAD

**Issue by Globegitter** [balderdashy/sails#2135]
When trying to set up a wildcard route, such as `'/*' : { view: 'index', skipAssets: true }`, Sails still seems to bind the default blueprint routes (e.g `/api/v1/forms/` in my case) but then serves the index.ejs anyway.

Looking at http://stackoverflow.com/questions/22770035/how-to-setup-sails-js-routes-to-support-pushstate-with-a-spa-on-the-frontend this behavior can be fixed by using `skipRegex`, which gives a solution but is a bit tedious and if you ever change your namespace/prefix, etc you have to remember updating `skipRegex`. It would be great to have some option to give default routes precedence over wildcard routes. Or maybe have some `skipBlueprints` setting or something that works similar to `skipRegex`, just taking out the 'manual' step of having to actually create/update the Regex.

**Comment by sailsbot** [comment]
Thanks for posting, @Globegitter.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
